src/notifications.c

```diff
@@ -334,12 +334,11 @@ void ReplicaBackupCallback(RedisModuleCtx *ctx, RedisModuleEvent eid, uint64_t s
 
 
 int CheckVersionForShortRead() {
-  // Minial versions: 6.2.5 or 6.0.15
-  if (redisVersion.majorVersion == 6) {
-    if (redisVersion.minorVersion == 2)
+  // Minial versions: 6.2.5

```

Comment:
-minor typo- (Minimal) Thx for the clear commit comment, now I know we need 6.2.5, soonish. We're already at 6.2.4, so no issues expected on our side. Cheers TW